.travis.yml

```diff
@@ -19,6 +19,8 @@ before_install:
   # There is a bug in travis. When using system ruby, bundler is not
   # installed and causes the default install action to fail.
   - sudo gem install bundler
+  - bundle install
+  - bundle exec pod repo update
 
 after_install:
   - ./bin/pod repo update --silent
```

Comment:
I'm not really sure.  I don't know what the story is with Travis-CI at all, tbh.  With the rest of the tweaks I needed to get done (and how long it took), after I got everything working, I just left it in.